features/projects/components/project-list/error-page/error-style.ts

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import { color, space, textFont } from "@styles/theme";
+import { Button } from "@features/ui";
+
+export const Container = styled.div`
+  border: 1px solid ${color("error", 300)};
+  border-radius: ${space(2)};
+  background: ${color("error", 25)};
+  padding: ${space(4, 0)};
+`;
+export const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-left: ${space(4)};
```

Comment:
Great that you use the utility functions and thus stick to the overall code conventions of the project.

Reply:
Now that I said this I realized this separate style file haha. If you want to create a separate file for your style I'd suggest a naming convention like `error.style.ts`. This is a bit more common I guess.

Reply:
oops. my bad! i do mean that way. will change it shortly, Thanks !!!